### ISSUE TITLE
fare frontend

### ISSUE THREAD

**Issue by xrmx** [xrmx/parse-medici-torino#3]
Una volta che un repository con i json è disponibile sarebbe comodo avere un frontend in js che consumi i dati. Di base dovrebbe:
- mostrare i punti su una mappa stradale di torino
- scegliere una data circoscrizione
- prendere la posizione dal device e centrare la mappa
- fare query sugli orari

**Comment by xrmx** [comment]
Potrebbe essere sufficiente usare un notebook observable:
https://observablehq.com/@pfoser/using-mapbox-gl-js-part-2

A prescindere dal frontend forse conviene fare il merge dei json in uno unico

**Comment by baldoalessandro** [comment]
Concordo sul json unico, mi sembra più fruibile e già c'è il campo `circoscrizione_numero` 👍

**Comment by xrmx** [comment]
Il geojson unico è https://github.com/xrmx/medici-asl-torino/blob/master/medici.geojson

**Comment by endorama** [comment]
Ciao! Ho due persone che stanno iniziando a programmare che vorrebbero provare a fare il FE :) Io faccio un po' di mentoring.

Sono @Amosgito e @MGF2, aspettatevi qualche PR nel prossimo futuro! 

@xrmx possiamo usare i dati direttamente da GitHub (raw.githubusercontent.com)? Hostiamo il FE nelle GitHub pages?

**Comment by xrmx** [comment]
Ciao @endorama, @Amosgito, @MGF2 ! Si prendete pure i dati da github, per quanto riguarda dove mettere il frontend van bene le gh pages. Avevo cominciato a farne uno su observable con mapbox ma mi son scornato con le api di mapbox e l'ho mollato lì.
Per quanto riguarda il geojson per ora ignorate il campo orari perchè non è documentato e non sono sicuro quel campo rimarrà così.
Buon lavoro!
